release-scripts/updateVersion.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const path = require("path");
+const packageJsonPath = path.resolve(process.cwd(), "package.json");
+const { name, version } = require(packageJsonPath);
+const fs = require("fs");
+
+const data = {
+    name: name,
+    version: version
+};
+
+const outputFilepath = path.resolve(process.cwd(), "src", "version.json");
+fs.writeFileSync(outputFilepath, JSON.stringify(data));
```

Comment:
Can you add some basic success and error handling?

Reply:
Done